buildenv/jenkins/jobs/infrastructure/signedOffByCheck

```diff
@@ -56,23 +56,28 @@ timeout(time: 6, unit: 'HOURS') {
                             def WRONG_COMMIT = true
                             def splitCommit = COMMIT.split("\\r?\\n")
                             def lastBlankLine = 0
-                            for (int i = splitCommit.size() - 1; i >= 0; i--) {
-                                if ( splitCommit[i] ==~ /^\s*$/ ) {
-                                    lastBlankLine = i
-                                    break  
-                                }
-                            }
-                            for (int j = splitCommit.size() - 1; j > lastBlankLine; j-- ) {
-                                if ( splitCommit[j] ==~ /\s*Signed\-off\-by\:\s+[^<]+\s+\<[^@]+@[^@]+\.[^@]+\>/ ) {
-                                    echo "The following commit appears to have a proper sign-off:\n${COMMIT}" 
-                                    WRONG_COMMIT = false
-                                    break
-                                }
-                            }
-                            if ( WRONG_COMMIT ) {
-                                echo "FAILURE - The following commit appears to have an incorrect sign-off:\n${COMMIT}"
-                                BAD_COMMITS << "${COMMIT}"
-                                FAIL = true
+                            if (splitCommit[splitCommit.size() - 1] ==~ /.*This reverts commit.*/){
```

Comment:
is `splitCommit[splitCommit.size() - 1]` the same as `splitCommit[-1]` ?

Reply:
fixed